content/en/apps/guides/hosting/4.x/app-developer.md

```diff
@@ -152,13 +152,13 @@ Always run the script from the directory where it lives.  If you launch it from
 Run the script with:
 
 ```
-./cht-docker-compose.sh
+sh ./cht-docker-compose.sh
```

Comment:
this will [override the shebang](https://linuxize.com/post/bash-shebang/#overriding-the-shebang), and we don't want to do that:

```suggestion
./cht-docker-compose.sh
```

Reply:
great, i see the interpreter is already defined here `#!/usr/bin/env bash`. thank you @mrjones-plip